src/api/addon.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Methods related to the DocumentCloud note API
+ */
+
+import session from "./session";
+import { Addon } from "@/structure/addon";
+import { apiUrl } from "./base";
+import { queryBuilder } from "@/util/url";
+import { grabAllPages } from "@/util/paginate";
+import { DEFAULT_EXPAND } from "./common";
+
+export async function getAddons(id, expand = DEFAULT_EXPAND, page) {
```

Comment:
The default expand does not make sense for add-ons.  I can explain how the expand parameter works.